Marlin/Marlin_main.cpp

```diff
@@ -3752,6 +3752,9 @@ inline void gcode_G28() {
     #if ENABLED(AUTO_BED_LEVELING_UBL)
       const bool bed_leveling_state_at_entry = ubl.state.active;
     #endif
+	#if ENABLED(MESH_BED_LEVELING)
+	  mbl.set_reactivate(mbl.active());
+	#endif
```

Comment:
Actually, turns out this would be bad. `G28` is specified to cause leveling to be turned off. Code that wants to do homing as part of its leveling procedure now has another method available: it can call `home_all_axes()` directly.